.travis.yml

```diff
@@ -108,6 +112,20 @@ before_script:
                   sudo make install
               )
           fi
+          if [ "$TRAVIS_OS_NAME" = linux -a "$BORINGSSL" ]; then
+            (git clone https://boringssl.googlesource.com/boringssl &&
```

Comment:
```bash
git clone --depth=1 …
```

… if you don't rewind to the latest tag or any blessed revision, using a shallow clone will be faster.

Reply:
good idea!